schema.graphql

```diff
@@ -230,6 +263,10 @@ input AddOpticalImageInput {
 
 type Query {
   dataset(id: String!): Dataset
+  user(id: String!): UserProfile
```

Comment:
@andy-d-palmer should user profiles be 'visible' only to the members of the groups/projects they belong to?

Reply:
Ideally, a user's name should only visible to other users that have view access to some of their datasets.